### PR TITLE
Customize fee

### LINKED ISSUE #79: Use /fee_stats endpoint

### PR DESCRIPTION
**Pull request by bartekn**
Display fee widget with fee recommendation when network is congested:

Screenshot 2019-03-19 at 13 28 57

Close #79.

### CODE REVIEW

**Review by morleyzhi — APPROVED** [review]
The changes make sense to me, 👍

**Review by vcarl — APPROVED** [review]

**Comment by tomquisel** [comment]
@bartekn I like the change! One tweak: maybe let users skip doing math in their head and report the fee in units of XLM? It also helps, because humans don't really understand units, and 5216 stroops sounds big. Reporting it as 0.0005216 XLM keeps it simple and in perspective.

**Comment by bartekn** [comment]
Good point, will change tomorrow.

File: controllers/send-widget.controller.es6 [hunk 1]
```diff
@@ -225,6 +262,16 @@ export default class SendWidgetController {
       this.amountAlertGroup.show(alert);
     }
 
+    // Check if fee is valid
+    if (this.fee < 100) {
+      let alert = new Alert({
+        title: '',
+        text: 'This fee is invalid.',
```
Comment (vogel) [inline controllers/send-widget.controller.es6]: "Minimum fee is 100" seems to better show the cause of error

**Comment by bartekn** [comment]
@tomquisel @vogel please take a look.

**Comment by bartekn** [comment]
> Reporting it as 0.0005216 XLM keeps it simple and in perspective.

@tomquisel after thinking about it more I started wondering if this option is also too complicated (as people have problems with very small numbers too: it's very hard to tell the difference between 0.00001 and 0.0001). Maybe we should really go with radio-like buttons: Low, Medium, High and display only recommendation and the value behind each option? Something like:

Screenshot 2019-03-20 at 20 51 05

Let me know what you think. Otherwise I'll just merge it in 24h from now.

**Comment by tomquisel** [comment]
@bartekn I still prefer just reporting it as a small XLM amount. If they can't tell the difference between multiple small numbers, that seems like the presentation leads the user to the right conclusion. The fee is really small, so don't worry about it. If we're showing them a number like 30000 stroops, the user is going to worry about it because it sounds big, which is the wrong conclusion.

**Pull request merged**